### PR TITLE
[MU4] Fix #316504: Update MusicXML Import of Fretboard Diagrams

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: https://musescore.org/en/node/316504

Port of #7455 to master

### CODE REVIEW

File: src/libmscore/fret.cpp [hunk 1]
```diff
@@ -1333,6 +1333,7 @@ void FretDiagram::writeMusicXML(XmlWriter& xml) const
             xml.tag("fret", "0");
             xml.etag();
         } else {
```
Comment (worldwideweary) [inline src/libmscore/fret.cpp:1335]: Careful, I explicitly removed this else statement for a reason in original PR :)

Comment (Jojo-Schmitz) [reply src/libmscore/fret.cpp:1335]: oops

Comment (Jojo-Schmitz) [reply src/libmscore/fret.cpp]: Better now?

**Comment by Jojo-Schmitz** [comment]
Currently no way to really test it, due to issue #7686

**Pull request merged**